views/partials/header.ejs

```diff
@@ -116,14 +116,14 @@
         <h1 class='heading-large'>
             Your application will time out soon
         </h1>
-        <p>We will reset your application if you do not respond in</p>
-        <p style="margin: 0; padding: 0 10px; float: left;">
+        <p class="govuk-body">We will reset your application if you do not respond in</p>
+        <p class="govuk-body" style="margin: 0; padding: 0 10px; float: left;">
```

Comment:
doesn't look like your code, but it's worth cleaning up the inline styles for peace of mind